UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/PlayFabMultiplayerAgentAPI.cs

```diff
@@ -163,7 +174,7 @@ internal static void SendHeartBeatRequest()
                 return;
             }
 
-            byte[] payloadBytes = Encoding.ASCII.GetBytes(payload);
+            byte[] payloadBytes = Encoding.UTF8.GetBytes(payload);
```

Comment:
nice catch

Reply:
This still isn't great but the JSON standard is the law. JSON5 has been announced and it's still UTF-8 so, I guess we stay here. Would be nice for the weirder strings to have UTF-32 but what can you do 😆